docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -443,6 +443,12 @@ So if you want to deploy your application somewhere, you need to copy the _runne
 
 NOTE: Before running the application, don't forget to stop the hot reload mode (hit `CTRL+C`), or you will have a port conflict.
 
+== Configuring the banner
```

Comment:
Also I need an anchor here. `banner` would be perfect.

Reply:
(Sorry a bit late but I'm working again on the announcement :))

Reply:
Ah yes! I'll do that :)

Reply:
I hope my very limited adoc knowledge was enough to imlement this requirement :P